libs/examples/src/chart/chart-examples.module.ts

```diff
@@ -38,28 +38,6 @@ import { DtExampleChartPie } from './chart-pie-example/chart-pie-example';
 import { DtExampleChartSelectionAreaDefault } from './chart-selection-area-default-example/chart-selection-area-default-example';
 import { DtExampleChartSinglePointData } from './chart-single-data-point-example/chart-single-data-point-example';
 import { DtExampleChartStream } from './chart-stream-example/chart-stream-example';
-import { DtExampleChartDonut } from './chart-donut-example/chart-donut-example';
-
-export const DT_CHART_EXAMPLES = [
```

Comment:
This will break the stackblitz example generations probably. 
@lukasholzer, can you please create an issue that this will break after you remove these constants?

Also, why do we only do this for the chart? Don't we need to do this for all other examples as well?

Reply:
Can I try the stackblitz examples somehow in this branch?

The constant was only needed to declare the components and put them into the entryComponents since angular9 with ivy no entryComponents are needed anymore.

You are right this can be done at all components. But please not in this PR.

Reply:
You can run `npm run shareable-examples`, this will generate the output that is being pushed to the examples repository in your dist folder. 

I'm questioning if the change is necessary, or if it would be sufficient for now to remove the entryComponents. If the other examples are broken if we do not do this, then please change them in this PR. otherwise, don't do anything...

Reply:
You can leave it as is and we will fix the shareable examples later on. You already had ideas on how to change the generation of the shared examples to be better.